src/sdg/api/authentication.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import AnonymousUser
+
+from rest_framework.authentication import TokenAuthentication
+
+from sdg.api.models import Token
+
+
+class UserlessTokenAuthentication(TokenAuthentication):
```

Comment:
```suggestion
from rest_framework.authentication import TokenAuthentication as _TokenAuthentication 

from sdg.api.models import Token


class TokenAuthentication(_TokenAuthentication):
```

This implies theres no user information, but there is, so please rename :)
The review below adds more user information.